api/apps/api/src/modules/specification/calculate-features/application/specification.repository.ts

```diff
@@ -0,0 +1,7 @@
+import { Specification } from '../../specification';
+
+export abstract class SpecificationRepository {
+  abstract save(specification: Specification): Promise<void>;
+
+  abstract getById(id: string): Promise<Specification | undefined>;
```

Comment:
it's a lot of duplication having a multiple specification repositories...

no extremes are healthy :D

Reply:
If it happens (seems like it does) they will be 100% same, will extract them once all application pieces are in place.

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/417/files via https://github.com/Vizzuality/marxan-cloud/pull/417/commits/94946b083a4e324560dcb1e72bdd94f5f4b46799